src/modules/core/router/containers/RouterRoutes.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+
+import RouterRoutes, {
```

Comment:
why don't join this logic with the other `RouterRoutes`? I think this abstraction is not needed

Reply:
So as I said, here we would call the `getRoutes` method and then pass it to the container.